eelbrain/_io/fiff.py

```diff
@@ -783,19 +783,14 @@ def sensor_dim(
             c_matrix, adj_ch_names = mne.channels.read_ch_adjacency(adjacency)
             # fix channel names
             if adjacency.startswith('neuromag'):
-                if ' ' not in adj_ch_names[0]:  # mne-python < ~1.2
+                if (' ' not in adj_ch_names[0]) and (' ' in ch_names[0]):  # mne-python < ~1.2
```

Comment:
Remove the comment, because it looks like it's inconsistent in some datasets, not just mne-Python
```suggestion
                if (' ' not in adj_ch_names[0]) and (' ' in ch_names[0]):
```